roles/wireguard/tasks/keys.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Delete the lock files
+  file:
+    dest: "/etc/wireguard/private_{{ item }}.lock"
+    state: absent
+  when: easyrsa_reinit_existent|bool == True
```

Comment:
WireGuard doesn't use easyrsa (it doesn't even use the rsa algorithm!), so you may want to rename this variable.

Reply:
Probably yes, but it's not related to this PR.

Reply:
:+1: